tests/ha/single_pod_restart/test_data_pod_restart.py

```diff
@@ -431,7 +389,7 @@ def test_deletes_after_pod_restart(self):
                                     'user_name': self.s3acc_name}}
         s3_test_obj = S3TestLib(access_key=access_key, secret_key=secret_key,
                                 endpoint_url=S3_CFG["s3_url"])
```

Comment:
Are we passing this object to put_get_delete calls?
This is mandatory argument right?